app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,9 @@
 <resources>
     <string name="app_name">House Organizer</string>
+    <string name="settings_speakable_text">Settings</string>
+    <string name="house_speakable_text">Home selection</string>
+    <string name="info_speakable_text">Info</string>
+    <string name="button_feedback_text">No button has been pressed</string>
     <string name="mainNameHint">Name</string>
```

Comment:
(Question) What do app_name and mainNameHint do ? 
(Notation) mainNameHint should probably be main_name_hint to follow the notation

Reply:
mainNainHint comes from the template. I will delete it now as well as other useless strings.
App name is the name of the app display at the top